client/webserver/site/src/js/markets.ts

```diff
@@ -1252,6 +1258,7 @@ export default class MarketsPage extends BasePage {
       })
       app().bindTooltips(div)
     }
+    Doc.setVis(unreadyOrders, page.unreadyOrdersMsg)
```

Comment:
I wonder if we'll want a way to indicate a problem even if the user is on a different market. Don't have to do it now. Could be a `db.ErrorLevel` notification maybe. Or maybe just well-placed messaging. Either way, I'd hate it if my orders on one market were not processing but I'm unaware because I'm looking at a different market.

Reply:
I was thinking to do a popup that shows up on the markets page after login in another PR.